kivy app/main.py

```diff
@@ -0,0 +1,124 @@
+# Filename: main.py
+from kivy.app import App
+from kivy.uix.widget import Widget
+from kivy.properties import NumericProperty, ReferenceListProperty,\
+    ObjectProperty
+from kivy.vector import Vector
+from kivy.clock import Clock
+from kivy.uix.popup import Popup
+from kivy.uix.label import Label
+from kivy.uix.button import Button
+import time
+from random import randrange, choice
+from kivy.config import Config
+Config.set('graphics', 'resizable', '0')
+Config.set('graphics', 'width', '480')
+Config.set('graphics', 'height', '640')  # Fixed window size for now
+
+# TO DO LIST:
+# Looking into using Kivy pages or screens (high score page, running page?)
+# If not using pages then menu screens/high score widgets
+# Placing trucks generated in array on screen
+# Add images/sprites
+# Change truck speeds
+# Leader board client server code
+# Add comments
+# End game function
+# Add timer, score change based on time taken
+
+popup = Popup(title='Welcome', content=Label(text='Instructions'), size_hint=(None, None), size=(400, 400))
+root = Widget()
+# Try and use pages have start page, a running page and a high score page
+
+
+class PlayerObject(Widget):
+    """ Planning to move PlayerObject into another file and import    """
+    score = NumericProperty(0)
+
+    def bounce_ball(self, ball):
```

Comment:
I can see that this is commented out, from my experience using pong as the base source code I have found that you probably don't need this definition at all.